.gitignore

```diff
@@ -9,7 +9,7 @@
 
 # IDE - VSCode
 .vscode/*
-!.vscode/settings.json
+.vscode/settings.json
```

Comment:
Why are we making this change? is it necessary? (this will be applied to all showcases and I do not think it to be needed.

Reply:
Hmm, I will comment it out.